idd/Energy+.idd.in

```diff
@@ -22727,9 +22726,9 @@ IndoorLivingWall,
   N2 , \field LED Nominal Intensity 
        \note The value represents photosynthetic photon flux density (PPFD) of LED grow light. 
        \note PPFD is measured in micro-mole per m2 per second (umol_m2s) which establishes exactly how many photosynthetically active radiation (PAR) photons are landing on a specific area.
-       \units umol_m2s
+       \units umol/m2-s
        \type real
-       \ip-units umol_m2s
+       \ip-units umol/ft2-s
```

Comment:
This change to using / instead of _ and using ft2 for the \ip-units should only impact IDF Editor